api/src/paths/project/{projectId}/funding-sources/{pfsId}/delete.test.ts

```diff
@@ -97,4 +110,52 @@ describe('delete a funding source', () => {
       expect(actualError.message).to.equal('Failed to build SQL delete statement');
     }
   });
+
+  it('should throw a 400 error when the delete fundingSource fails, because the response has no rows', async () => {
```

Comment:
why is this test failing? it says when response has no rows but you are returning a non null rows value